Library/Formula/tippecanoe.rb

```diff
@@ -19,12 +19,10 @@ def install
   end
 
   test do
-    path = testpath/"test.json"
-    path.write <<-EOS.undent
+    (testpath/"test.json").write <<-EOS.undent
       {"type":"Feature","properties":{},"geometry":{"type":"Point","coordinates":[0,0]}}
     EOS
-    output = `#{bin}/tippecanoe -o test.mbtiles #{path}`.strip
-    assert_equal 0, $?.exitstatus
-    assert_equal "using layer 0 name test", output
+    safe_system "#{bin}/tippecanoe", "-o", "test.mbtiles", "test.json"
```

Comment:
To be honest I'm not fussed so much about `"#{bin}"` Vs `bin`. I think the latter looks cleaner but both are just as easy to understand the logic of.

I'm more picky about being consistent with the syntax we use, more than anything, so we don't end up with a confusing hodgepodge.